shard-operator-parent/shard-operator-core/src/main/java/com/redhat/service/smartevents/shard/operator/core/resources/CustomResourceStatus.java

```diff
@@ -40,35 +40,35 @@ public final Optional<Condition> getConditionByType(final String conditionType)
 
     @JsonIgnore
     public final boolean isReady() { // Don't use this in V2 as READY constant doesn't exist https://issues.redhat.com/browse/MGDOBR-1369
-        return conditions.stream().anyMatch(c -> ConditionTypeConstants.READY.equals(c.getType()) && ConditionStatus.True.equals(c.getStatus()));
+        return conditions.stream().anyMatch(c -> ConditionTypeConstants.READY.equals(c.getType()) && getTrue().equals(c.getStatus()));
```

Comment:
This is an abstract class in `-core` so I cannot use any _FIXED_ values.

Access is delegated to the different implementations so the correct version-specific literal can be used.